### DISCUSSION TITLE
EODS+ 2.0 and more mods

### DISCUSSION THREAD

**Discussion by Astorixim** [Foxlider/FASTER#161] [Q&A]
Hello, I just try up with start new server for clan members and i tried add our modpack which we are using but few mods as EODS+2.0 server set as EODS_2_0 but arma still noted as EODS+ 2.0 so the server throw error and disconnect us. Have any one same tips what to do? I tried youtube... local mods... URL steam of that mod. But nothink helped. Thanks for your help.

**Comment by MildlyInterested** [comment]
The mod name does not matter, what's the error the server is throwing?
